locale/README.md

```diff
@@ -0,0 +1,24 @@
+# HOWTO translate GRASS GIS messages
+
+## Where to translate user message
+
+Translators work in <https://weblate.osgeo.org/projects/grass-gis/>.
+Messages translated in Weblate then will be automatically submitted to the
+GRASS GIS source repo as pull requests.
+
+If you have any difficulties please ask for help
+on the GRASS development mailing list `grass-dev lists.osgeo.org`
+(subscribe at <https://lists.osgeo.org/mailman/listinfo/grass-dev>). Your
+willingness, time, and effort translating are valuable.
+
+Note that GRASS must be configured with `--with-nls` and (re)compiled
+to make use of the translated messages.
+
+PLEASE HELP TO TRANSLATE GRASS MESSAGES!
```

Comment:
```suggestion
**Please help us translate all the GRASS messages!!**
```

Reply:
Caps are usually interpreted as shouting, to highlight text I suggest to use bold face.